requirements-dev.txt

```diff
@@ -13,7 +13,7 @@
 -e .
 astroid
 autopep8
-black==19.10b0
```

Comment:
Why will we touch black here?

Also, is above changed by black?

Reply:
Was a hefty rabbit hole, lints were failing so I just ran black on the offending files, was using my own black installation because I hadn't found `make black` yet, had a different line width setting to the one in pyproject  and thought lints were failing because of mismatched black versions so I did a pip --upgrade. Figured out what was going on eventually, already had files formatted with a newer black and figured I'd bring that up to date and run it on the rest of the files in this pull since I was already having to change a bunch of code to pass flake checks and test cases. A hefty amount of the above code is modified by black, but there's also changes I made to code that I think was added while CI was down because they didn't pass other test cases. I can separate the black version bump out I guess, probably going to take a hot minute.

Reply:
Got it, I have two thought about this,

We can neither migrate to GitHub Action first (adding `.github/workflows/ci.yml`, remove `.travis.yml`), then do linting fix.
Or, fix linting first, then migrate to GitHub Action.

Just don't mix them into same commit/PR.

Reply:
Alright, scooped out all the black changes. Presumably we'd be doing the actions migration first, since you said you wanted CI before we start merging things.